ArjunaCore/arjuna/classes/com/arjuna/ats/arjuna/coordinator/BasicAction.java

```diff
@@ -3386,6 +3386,19 @@ private final boolean checkChildren (boolean isCommit)
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_57(get_uid());
                 } else {
                     tsLogger.i18NLogger.warn_coordinator_BasicAction_58(get_uid());
+
```

Comment:
Well, this information needs to be printed when the level includes warn (i.e most times) - if I guard the concat with a trace check then the info won't be printed unless trace is on (unlikely).

Maybe I am speaking to another area of the code as your comment currently applies to the blank 3389 - are you talking about the code in lines 3394 - 3399?